### ISSUE TITLE
Unit Tests should be abortable

### ISSUE THREAD

**Issue by retailcoder** [rubberduck-vba/Rubberduck#274]
The next logical step after #273 - the _Test Explorer_ window's toolbar needs to feature a "Stop" (/abort) button that automates breaking out of, and stopping the running VBA code; to keep things simple, an aborted session should result in all tests being considered "aborted" and thus, if a test was completed, its result is changed to "aborted" regardless of whether it succeeded or not.

The commandbar button ID for the "Break" button is 189; the ID for the "Stop" button is 228; both are on the "Standard" commandbar.

**Comment by Hosch250** [comment]
While I'm in this, I'll take a look, but I can't guarantee anything.

**Comment by retailcoder** [comment]
Fun facts: there is no "aborted" test result.

Note: firing command 228 for the currently running test and making it "aborted" won't be enough, since the run loop is going to call application.run for the remaining tests; you need to break out of that loop to abort the whole run.

**Comment by Hosch250** [comment]
@retailcoder R# doesn't mark them as aborted.  It marks the unrun ones as Unknown, and the rest as the result of the run.

**Comment by Hosch250** [comment]
I have this sort-of done, but I can't get it to let me click the button while they are running.  They entire test explorer UI locks up...

**Comment by comintern** [comment]
Closing in favor of the duplicate #3158.
